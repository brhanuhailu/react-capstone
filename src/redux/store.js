import { configureStore } from '@reduxjs/toolkit';
import cryptoSlice from './crypto/cryptoSlice';

const store = configureStore({
  reducer: {
    cryptoData: cryptoSlice,
  },
});

export default store;
