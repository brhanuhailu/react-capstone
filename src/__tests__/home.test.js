import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import Home from '../components/home';
import store from '../redux/store';

describe('Home', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <BrowserRouter>
            <Home />
          </BrowserRouter>
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
