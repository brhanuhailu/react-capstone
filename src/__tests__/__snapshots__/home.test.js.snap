// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should renders home page correctly 1`] = `
<header
  className="coin-container"
>
  <div
    className="home-serch"
  >
    <input
      onChange={[Function]}
      placeholder="Search Coin..."
      type="text"
      value=""
    />
  </div>
  <div
    className="home-list"
  >
    <h1>
      No related coins found!
    </h1>
  </div>
</header>
`;
